src/main/java/org/ladysnake/blabber/impl/common/model/DialogueIllustrationItem.java

```diff
@@ -0,0 +1,54 @@
+package org.ladysnake.blabber.impl.common.model;
```

Comment:
Should the illustrations package be in model? They get serialized, but they're more "implementations" rather than part of the actual dialogue model, the model part of them is in the API.

Reply:
Good question, I think I would put them outside of model.